Tests/Exporter/ExporterTest.php

```diff
@@ -16,17 +16,21 @@
 
 class ExporterTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * @expectedException RuntimeException
-     */
     public function testFilter()
     {
+        $this->setExpectedException('RuntimeException', 'Invalid "foo" format');
```

Comment:
I slightly improved the error message, and wanted to test that. I can make it a separate commit if necessary, and that will be cumbersome, but a separate PR for that would be too much.